python/pyspark/ml/feature.py

```diff
@@ -347,6 +353,28 @@ class Bucketizer(JavaTransformer, HasInputCol, HasOutputCol, HasHandleInvalid,
     >>> bucketed = bucketizer.setHandleInvalid("skip").transform(df).collect()
     >>> len(bucketed)
     4
+    >>> bucketizer2 = Bucketizer(splitsArray=
+    ...     [[-float("inf"), 0.5, 1.4, float("inf")], [-float("inf"), 0.5, float("inf")]],
+    ...     inputCols=["values", "numbers"], outputCols=["buckets1", "buckets2"])
+    >>> bucketed2 = bucketizer2.setHandleInvalid("keep").transform(df).collect()
+    >>> len(bucketed2)
+    6
+    >>> bucketed2[0].buckets1
```

Comment:
Perhaps it would be cleaner to do a `df.show()` here? Likewise above for `bucketed` we could change that part of the doctest too.